Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingImageBufferBackend.mm

```diff
@@ -103,11 +105,15 @@ static CFDictionaryRef makeContextOptions(const DynamicContentScalingImageBuffer
 
     RetainPtr<NSDictionary> options;
     RetainPtr<NSMutableArray> ports;
+    RetainPtr<CAMachPortRef> caMachPort;
     if (m_resourceCache) {
+        auto machSendRight = MachSendRight::create(m_resourceOwner.taskIdToken());
+        caMachPort = CAMachPortCreate(machSendRight.sendRight());
```

Comment:
I think I want `leakSendRight` instead of `sendRight` here, will need to test once the other change merges